src/etc/inc/system.inc

```diff
@@ -2042,6 +2042,11 @@ function system_dmesg_save() {
 
 function system_set_harddisk_standby() {
 	global $g, $config;
+	// Get only suitable candidates for standby; using get_smart_drive_list()
+	// from utils.inc to get the list of drives.
+	$harddisks = array();
+	$harddisks = get_smart_drive_list();
```

Comment:
This variable is only used if $config['system']['harddiskstandby'] is set, then this declaration can be moved inside that if()